### PR TITLE
Style:Logos-added-register-page-#292

### PR DESCRIPTION
**Pull request by Surya-V-28**
made the changes on the previous PR #313 ,
image

Please take a look at the PR . the issues PR #292 
I am open to any suggestion to make it more better, Let me know if there are any issues with it

@anfibiacreativa , @juliamuiruri4 ,@manekinekko, @sinedied

### CODE REVIEW

File: packages/portal/src/index.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
     />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (sinedied) [inline packages/portal/src/index.html:15]: This should not be needed as you're using font awesome icons, not the material ones

**Review by sinedied — CHANGES_REQUESTED** [review]
LGTM, except for the small comment

File: packages/portal/src/index.html (hunk 1 above)
Comment (Surya-V-28) [reply packages/portal/src/index.html:15]: Yes, you can remove it, if its not need or do you want me to remove it and make a new PR ? @sinedied

**Comment by manekinekko** [comment]
Hi @Surya-V-28 any updates for this PR?

**Comment by Surya-V-28** [comment]
@manekinekko , I think it has no issues to merge , I have made the requested changes if any new needed let me know, i think its okay to merge

**Review by manekinekko — DISMISSED** [review]

**Comment by manekinekko** [comment]
Merged! Thank you @Surya-V-28!

**Pull request merged**